astro/src/components/SearchForm.astro

```diff
@@ -0,0 +1,63 @@
+---
+---
+
+<form
+  enctype="multipart/form-data"
+  action="/api/input/form"
+  method="post"
+  class="border-2 border-color: #dfe1e5; shadow-xl rounded-full flex justify-center items-center pr-3 pl-3"
```

Comment:
`border-color: #dfe1e5` is a css style, added in a class attribute @iam-benyamin